modules/apps/commerce/commerce-product-definitions-web/src/main/resources/META-INF/resources/js/edit_cp_definition_visibility.js

```diff
@@ -29,16 +28,13 @@ function handleEvent({
 
 	return AdminCatalogResource.updateProduct(productId, formattedData).finally(
 		() => {
-			Liferay.fire(commerceEvents.FDS_UPDATE_DISPLAY, {
-				id: dataSetId,
-			});
+			sessionStorage.setItem(
+				'com.liferay.commerce.product.definitions.web.successMessage',
+				Liferay.Language.get('your-request-completed-successfully'),
+				sessionStorage.TYPES.NECESSARY
+			);
```

Comment:
@gianmarcobrunialti here the other logic

Also: but this is not something related to you changes @dsitu : I think we are doing this operation (refresh + toast with success message) also if the updateProduct promise fails

Reply:
@smottal some changes to fix error checking 4ee6c4a6553ced05713251a65e657a2d722b3be1